### PR TITLE
feat: export keepalived metrics from keepalived container

### PR DESCRIPTION
**Pull request by clwluvw**
introduced `--container-name` to use container mode

### CODE REVIEW

**Review by mehdy — CHANGES_REQUESTED** [review]
As we discussed.

**Comment by clwluvw** [comment]
@mehdy PTAL

File: internal/types/host/keepalived_host_collector_host.go [hunk 1]
```diff
@@ -30,7 +30,7 @@ type KeepalivedHostCollectorHost struct {
 }
 
 // NewKeepalivedHostCollectorHost is creating new instance of KeepalivedHostCollectorHost
-func NewKeepalivedHostCollectorHost(useJSON bool, pidPath string) *KeepalivedHostCollectorHost {
+func NewKeepalivedHostCollectorHost(useJSON bool, pidPath string) collector.Collector {
```
Comment (mehdy) [inline internal/types/host/keepalived_host_collector_host.go:33]: Why should we change the return type to an interface?

Comment (clwluvw) [reply internal/types/host/keepalived_host_collector_host.go]: No need. That was my fault :D

**Review by mehdy — APPROVED** [review]

**Pull request merged**